chargeOclock/Source/Models/Client.swift

```diff
@@ -32,6 +32,12 @@ class Client {
 			completion(error)
 		}
 	}
+
+	static func addClient(name: String, completion: @escaping (Error?, Data?) -> Void) {
```

Comment:
you'll also need to change the constructor, so the id can't be set from the (non-JSON) constructor... 😉